module/zfs/zfs_chksum.c

```diff
@@ -93,7 +101,22 @@ chksum_stat_kstat_headers(char *buf, size_t size)
 }
 
 static int
-chksum_stat_kstat_data(char *buf, size_t size, void *data)
+chksum_kstat_headers_mini(char *buf, size_t size)
+{
+	ssize_t off = 0;
+
+	off += snprintf(buf + off, size, "%-23s", "implementation");
+	off += snprintf(buf + off, size - off, "%8s", "1k");
+	off += snprintf(buf + off, size - off, "%8s", "4k");
+	off += snprintf(buf + off, size - off, "%8s", "16k");
+	off += snprintf(buf + off, size - off, "%8s", "64k");
+	(void) snprintf(buf + off, size - off, "%8s", "256k\n");
+
+	return (0);
+}
```

Comment:
Good thought, although always printing the full header with 0's for tests which weren't run would make this a little easier to parse.